### PR TITLE
Feature/handle jwt expire on frontend

### LINKED ISSUE #145: Handle token expiration gracefully on the front-end
When #144 is implemented we should detect that on the front-end and redirect to login page.
We should handle this both for management and demo front-end.

### PR DESCRIPTION
**Pull request by Jakedavies**
closes #145

### CODE REVIEW

File: admin/src/helpers/internal-fetch.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { merge } from 'lodash/object';
+import session from './Session';
+
+const internalFetch = (url, config = {}) => {
```
Comment (maksymilian-majer) [inline admin/src/helpers/internal-fetch.js]: @Jakedavies after you rebase can you refactor to use internalFetch on the changes I submitted?

File: admin/src/helpers/internal-fetch.spec.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    });
+
+    const res = await internalFetch('');
+    expect(res.ok).toEqual(true);
+    expect(res.status).toEqual(200);
+    expect(await res.json()).toEqual({ test: 'test' });
+  });
+
+  it('Should redirect and destroy the session if JWT is expired', async () => {
+    session.getToken = jest.fn().mockImplementationOnce(() => 'token');
```
Comment (maksymilian-majer) [inline admin/src/helpers/internal-fetch.spec.js]: I'm mocking session using `jest.mock('../../helpers/Session');` in other places, but it's ok too :)

File: cg/src/utils/jwt.js [hunk 3]
```diff
@@ -42,7 +42,7 @@ exports.managementJWT = new JWT(
   'HS256',
   process.env.MANAGEMENT_SECRET,
   process.env.MANAGEMENT_SECRET,
-  { expiresIn: getTokenExpiry() }
+  { expiresIn: 5 }
```
Comment (maksymilian-majer) [inline cg/src/utils/jwt.js]: I think you intended this for testing purposes only and should be reverted back to `getTokenExpiry()`?

Comment (fjcero) [reply cg/src/utils/jwt.js]: Can we define an env file for testing or doesn’t make sense?

Comment (maksymilian-majer) [reply cg/src/utils/jwt.js]: I believe there is a default for token expiry, so nothing needs to be done for this setting

File: admin/src/helpers/internal-fetch.js (hunk 1 above)
Comment (Jakedavies) [reply admin/src/helpers/internal-fetch.js]: will do

File: cg/src/utils/jwt.js (hunk 3 above)
Comment (Jakedavies) [reply cg/src/utils/jwt.js]: i did not mean to change this. will revert

**Pull request merged**